### DISCUSSION TITLE
关于alist新增学习通云盘的idea

### DISCUSSION THREAD

**Discussion by ShiDai567** [AlistGo/alist#6634] [Ideas]
![mmexport1718768266425.jpg](https://github.com/alist-org/alist/assets/97817186/b87436c1-dbf7-43e6-8c5e-406373fa9f26)

学习通上也有一个云盘，求作者能弄进去吗

**Comment by iwvw** [comment]
> ![mmexport1718768266425.jpg](https://private-user-images.githubusercontent.com/97817186/340904024-b87436c1-dbf7-43e6-8c5e-406373fa9f26.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5NzIyNTUsIm5iZiI6MTcxODk3MTk1NSwicGF0aCI6Ii85NzgxNzE4Ni8zNDA5MDQwMjQtYjg3NDM2YzEtZGJmNy00M2U2LThjNWUtNDA2MzczZmE5ZjI2LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDEyMTIzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUwYzc1ODAxNjY4NWY1NmQ1NzY5YzZhNTRlNjNlY2M3MGE5YjdmNzA5MDA5NzgwM2E3N2M2NWFmNThlMzhiNGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Lvd4swW9OegEkwlgw7cSs30lBx3Gdx-knsv1_qxnJNY)
> 
> 学习通上也有一个云盘，求作者能弄进去吗

早都有了，你看看文档
